
发送请求
连接失败，重试
发送请求
连接失败，尝试重启
开始重启
尝试获取端口
端口获取成功tcp      129      0 0.0.0.0:8000            0.0.0.0:*               LISTEN      1522/python3        
tcp        0      0 10.9.185.1:51286        117.50.23.70:8000       ESTABLISHED 51226/python3       
tcp        0      0 10.9.185.1:51426        117.50.23.70:8000       ESTABLISHED 55045/python3       
tcp        0      0 10.9.185.1:51410        117.50.23.70:8000       ESTABLISHED 54703/python3       
tcp        0      0 10.9.185.1:51326        117.50.23.70:8000       ESTABLISHED 52422/python3       
tcp        0      0 10.9.185.1:51398        117.50.23.70:8000       ESTABLISHED 54253/python3       
tcp        0      0 10.9.185.1:51314        117.50.23.70:8000       ESTABLISHED 51974/python3       
tcp        0      0 10.9.185.1:8000         223.104.170.83:13296    ESTABLISHED 1522/python3        
tcp        0      0 10.9.185.1:51484        117.50.23.70:8000       ESTABLISHED 56463/python3       
tcp        0      0 10.9.185.1:51438        117.50.23.70:8000       ESTABLISHED 55425/python3       
tcp        0      0 10.9.185.1:51354        117.50.23.70:8000       ESTABLISHED 53207/python3       
tcp        0      0 10.9.185.1:51466        117.50.23.70:8000       ESTABLISHED 56171/python3       
tcp        0      0 10.9.185.1:51186        117.50.23.70:8000       ESTABLISHED 48626/python3       
tcp        0      0 10.9.185.1:51382        117.50.23.70:8000       ESTABLISHED 53965/python3       
tcp        0      0 10.9.185.1:51514        117.50.23.70:8000       ESTABLISHED 57253/python3       
tcp        0      0 10.9.185.1:51454        117.50.23.70:8000       ESTABLISHED 55711/python3       
tcp        0      0 10.9.185.1:51258        117.50.23.70:8000       ESTABLISHED 50476/python3       
tcp        0      0 10.9.185.1:51230        117.50.23.70:8000       ESTABLISHED 49682/python3       
tcp        0      0 10.9.185.1:51370        117.50.23.70:8000       ESTABLISHED 53505/python3       
tcp        0      0 10.9.185.1:51214        117.50.23.70:8000       ESTABLISHED 49385/python3       

sudo kill -9 1522
执行杀掉
sudo kill -9 51226
执行杀掉
sudo kill -9 55045
执行杀掉
sudo kill -9 54703
执行杀掉
sudo kill -9 52422
执行杀掉
sudo kill -9 54253
执行杀掉
sudo kill -9 51974
执行杀掉
sudo kill -9 1522
执行杀掉kill: (1522): No such process

sudo kill -9 56463
执行杀掉
sudo kill -9 55425
执行杀掉kill: (55425): No such process

sudo kill -9 53207
执行杀掉kill: (53207): No such process

超时
移动日志
重启命令发送
程序退出